src/main/java/com/chen/itproject/util/JwtUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.chen.itproject.util;
+
+import io.jsonwebtoken.*;
+import org.springframework.stereotype.Component;
+
```

Comment:
Maybe remove the extra line here for better formatting

Reply:
got it